web/src/store/index.ts

```diff
@@ -40,6 +42,7 @@ export const useStore = create<StoreState>(() => ({
   passcode: '',
   autolock: 0,
   items: [{ name: '', metadata: '', remove: false }],
+  characters: [],
```

Comment:
Should add an empty string to the initial state array so that the initial field is created to match the behaviour of others.

Reply:
Done.